_posts/2021-05-04-stability.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: Dask User Summit 2021
+author: Matthew Rocklin
+theme: twitter
+---
+{% include JB/setup %}
+
+Dask is moving fast these days.  Sometimes we break things as a result.
+
+Historically this hasn't been a problem, according to our survey last year
+most users were fairly happy with Dask's stability.
+
+<img src="/images/2020_survey/2020_27_0.png">
+
+However the last year has seen a lot of evolution of the project,
+which in turn causes code churn.
+This can cause friction for downstream users today,
+but also means more-than-incremental changes for the future.
+We've optimized a little bit for long-term growth over short-term stability.
+
+
+### Motivation for change
+
+There are two structural things driving some of these changes:
+
+1.  An increase in computational scale
+2.  An increase in organizational scale
+
+
+### Computational Scale
+
+Dask today is used across a wider range of problems,
+a more diverse set of hardware,
+and at larger scales more routinely than before.
+
+Addressing this increase in scale across many dimensions has caused us to
+redesign Dask's internal infrastructure in several ways.
```

Comment:
These changes are really interesting. If I could, I'd dig in to this section and look at the relevant PRs (benchmarks/timings would be especially interesting).

What are some benefits of these changes? What are some use cases where those benefits appear? This section really only mentions the negative impacts (downstream bugs).

Reply:
This is a good suggestion. However, I don't think we should hold this post back because of it - we can make another blogpost that goes into more of these nitty gritty details later on.

Reply:
Thanks! I'm curious to read that post.